src/test/java/com/hocs/test/glue/decs/SearchStepDefs.java

```diff
@@ -26,6 +26,7 @@
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import java.awt.Checkbox;
```

Comment:
are you using this package in the automation, if its unused then remove the import